### DISCUSSION TITLE
Switch to light background

### DISCUSSION THREAD

**Discussion by txtyash** [helix-editor/helix#857] [Q&A] [answered]
How to switch to light background. Changing theme just has dark backgrounds.

**Comment by archseer** [comment, marked as answer]
Most themes are dark but you can use Ingrid which is a light theme. https://github.com/helix-editor/helix/wiki/Themes

**Reply by txtyash** [reply]
> Most themes are dark but you can use Ingrid which is a light theme. https://github.com/helix-editor/helix/wiki/Themes

Ingrid is a bad theme. Is there no other light theme. I tried all of em. Maybe there isn't.

**Reply by archseer** [reply]
I find this a bit mean to the original contributor that took the time and effort to port the Ingrid theme.

All themes under `runtime/themes` are user contributed, you can port your preferred theme using [the docs here](https://docs.helix-editor.com/themes.html).

**Comment by archseer** [comment]
What's your current theme elsewhere? It should be easy to port :)

**Reply by txtyash** [reply]
> What's your current theme elsewhere? It should be easy to port :)

" Themes light and dark both supported.
Plug 'morhetz/gruvbox'
Plug 'nightsense/snow' 
Plug 'cocopon/iceberg.vim' 
Plug 'junegunn/seoul256.vim'
Plug 'nightsense/stellarized' 
Plug 'nightsense/cosmic_latte' 
Plug 'NLKNguyen/papercolor-theme' 
Plug 'chriskempson/vim-tomorrow-theme'

**Reply by txtyash** [reply]
Anyways It is a nice idea to keep themes inside the editor itself. I hope y'all keep only a few, nice, eye-friendly themes. Plugins really make a mess.

**Reply by pickfire** [reply]
Currently the themes are written by contributors themselves, so if you see a theme in the contrib, it's probably used by someone. Maybe you can try writing a theme, for me at least one of the theme is good enough for me.
